workspaces/blackduck/plugins/blackduck-backend/src/api/BlackDuckRestApi.ts

```diff
@@ -54,7 +54,7 @@ export class BlackDuckRestApi {
   }
   public async getProjects(name: string): Promise<BD_REST_API_RESPONSE> {
     const projects = await fetch(
-      `${this.host}/projects?limit=999&q=name:${name}`,
+      `${this.host}/projects?limit=999&q=${encodeURIComponent(`name:${name}`)}`,
```

Comment:
Just thinking wide, can we have this `encodeURIComponent` function on fetching the version too?